java/src/com/ibm/streamsx/topology/function/FunctionContext.java

```diff
@@ -0,0 +1,145 @@
+package com.ibm.streamsx.topology.function;
+
+import java.net.MalformedURLException;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.ThreadFactory;
+
+import com.ibm.streams.operator.ProcessingElement;
+
+/**
+ * Context for a function executing in a IBM Streams application.
+ */
+public interface FunctionContext {
+    
+    /**
+     * Get the processing element hosting a function 
+     * @return ProcessingElement hosting a function.
+     */
+    ProcessingElement getPE();
```

Comment:
Not sure we need to, the api elsewhere depends on the Java Operator API.

Do you think some of the ProcessingElement API's need to be hidden?